.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[build]
+rustflags = ["--cfg", "tokio_unstable"]
```

Comment:
This was for running with tokio-console, should probably be removed before merging